README.md

```diff
@@ -7,6 +7,59 @@ Node-API contract from Node.js allows consumers of this library to interact with
 JavaScript engine with a consistent interface. This library also provides some optional
 polyfills that consumers can include if required.
 
+
+## **Building - All Development Platforms**
+
+**Required Tools:** [git](https://git-scm.com/), [CMake](https://cmake.org/), [node.js](https://nodejs.org/en/)
+
+The first step for all development environments and targets is to clone this repository. 
+
+Use a git-enabled terminal to follow the steps below. The `--recursive` flag is necessary as 
+Babylon Native makes extensive use of submodules for dependencies.
+
+```
+git clone --recursive https://github.com/BabylonJS/JsRuntimeHost.git
+```
+
+Babylon Native requires Babylon.js. You will need to install NPM packages to resolve these dependencies. From the root of the `JsRuntimeHost` 
+repository on the command line, run the following commands:
+
+```
+cd Tests
+npm install
+```
+
+
+## **Building on Windows, Targeting Android**
+
+_Follow the steps from [All Development Platforms](#all-development-platforms) before proceeding._
+
+**Required Tools:**
+[Android Studio](https://developer.android.com/studio), [Node.js](https://nodejs.org/en/download/), [Ninja](https://ninja-build.org/)
+
+The minimal requirement target is Android 5.0.
+
+Only building with Android Studio is supported. CMake is not used directly. Instead, Gradle
+is used for building and CMake is automatically invocated for building the native part.
+An `.apk` that can be executed on your device or simulator is the output.
+
+
+First, download the latest release of Ninja, extract the binary, and add it to your system path.
+
+Once you have Android Studio downloaded, open it in Administrator Mode. Then, you need to set up an Android emulator if you do not have a physical Android device. You can do this by selecting `Tools` -> `Device Manager` and then selecting a device. (We are using Pixel 2 API 27). 
```

Comment:
> open it in Administrator Mode

I don't think this is necessary?

Reply:
It does seem to be necessary the first time you build.